src/util.js

```diff
@@ -148,3 +148,25 @@ export function splitCommas (value) {
 
 	return ret;
 }
+
+/**
+ * Check if a CSS custom property is registered.
+ * @param {string} property - The property to check.
+ * @param {Window} [window] - The window to check in.
+ * @returns {boolean} Whether the property is registered.
+ */
+export function isRegisteredProperty (property, window = globalThis) {
+	let document = window.document;
+
+	dummy ??= document.createElement("div");
+	if (!dummy.parentNode) {
```

Comment:
What is the purpose of this check? When would it not be true, since we remove it at the end of this function and it's a sync function?

Reply:
When we move this function, we won't need this check.